### PR TITLE
add latest speedtest endpoint

### PR DESCRIPTION
**Pull request by allenan**

### CODE REVIEW

File: src/helpers/mobileApi.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      'Cache-Control': 'no-cache',
+      'Content-Type': 'application/json',
+    },
+  })
+
+  if (!response.ok) {
+    throw { status: response.status, message: response.statusText }
+  }
+
+  const text = await response.text()
```
Comment (tyler-whitman) [inline src/helpers/mobileApi.js:27]: Can this `await response.text()` throw too?

**Review by tyler-whitman — APPROVED** [review]
Looks good, just some minor questions

File: src/helpers/mobileApi.js (hunk 1 above)
Comment (allenan) [reply src/helpers/mobileApi.js:27]: I copied this from the `cellHotspots.js` file you wrote 3 months ago, so I'm not sure, can it? 😆

Comment (tyler-whitman) [reply src/helpers/mobileApi.js:27]: Ah ok I think I got that mostly from one of the other ones we have haha. Seems like its been working fine, so I would say its good to go.

**Pull request merged**